### PR TITLE
guard events

### PR DESCRIPTION
**Pull request by fsansalvadore**

### CODE REVIEW

File: apps/www/pages/_app.tsx [hunk 1]
```diff
@@ -2,53 +2,47 @@ import '@code-hike/mdx/styles'
...
+      {
+        category: 'www',
+        action: 'page_views',
+        label: url,
       },
-    })
+      telemetryProps,
+      router
+    )
   }
```
Comment (phamhieu) [inline apps/www/pages/_app.tsx:43]: why converting from `telemetry/page` to `/telemetry/event`?

Comment (phamhieu) [reply apps/www/pages/_app.tsx:43]: on studio, we still use `/telemetry/page` https://github.com/supabase/supabase/blob/cd55df330df8d9415c7ff183aa92538c14c32e06/apps/studio/components/ui/PageTelemetry.tsx#L80-L90

Comment (phamhieu) [reply apps/www/pages/_app.tsx:43]: `/telemetry/page` and `/telemetry/event` are different. I don't think we can create a page_view event to replace the call to `/telemetry/page`."

Comment (fsansalvadore) [reply apps/www/pages/_app.tsx:43]: Ah you're right. I'm gonna revert this "refactoring" on the `www/_app` then.

**Review by pamelachia — APPROVED** [review]

**Review by 4L3k51 — APPROVED** [review]

**Pull request merged**